README.md

```diff
@@ -91,10 +91,34 @@ config.keys = {
 }
 ```
 
-4. Optional, enable encryption (recommended):
+4. Delete a saved state file via. fuzzy finder:
```

Comment:
I don't think this should be a part of the example setup as it should be a quick and easy way for users to read and test the basic functionality and I think that deleting the state is a bit overkill for this section of the readme.

Can you move it to a seperate section, under configuration maybe? 😃